block/block.md

```diff
@@ -1 +1,95 @@
 # Block
+
+## Abstract
+
+The block manager is a key component of full nodes (including sequencer) that is responsible for block production (in case of sequencer full node) or block syncing (syncing in this context includes retrieving the published blocks from the network, validating them to raise fraud proofs upon validation failure, and updating the state and storing the validated blocks). A full node invokes multiple of the block manager functionalities in parallel, such as:
+
+* block production (in case of sequencer full nodes)
+* block publication to DA network
+* block retrieval from DA network
+* block retrieval from Blockstore (which retrieves blocks from the p2p network)
+* block syncing
+
+## Protocol/Component Description
+
+The block manager is initialized using several parameters as defined below:
+
+* `signing key`: used for signing the blocks after production
+* `config`: block manager configurations
+* `genesis`: initialize the block manager with genesis state
+* `localstore`: store blocks and states
```

Comment:
is this a relative path? absolute path?

Reply:
is not a path. its a db. added additional information.